packages/desktop-client/src/components/FinancesApp.tsx

```diff
@@ -260,6 +260,10 @@ export function FinancesApp() {
                   path="/rules"
                   element={<NarrowAlternate name="Rules" />}
                 />
+                <Route
+                  path="/rules/edit/:id?"
```

Comment:
Can this be to match the convention for other routes?
```suggestion
                  path="/rules/:id"
```